compiler/shared_ast/print.ml

```diff
@@ -457,7 +460,8 @@ let rec expr_aux :
       Cli.format_with_style [List.hd colors] fmt ")")
     else expr colors fmt e1
   in
-  let lhs ex = paren ~rhs:false ex in
+  let default_punct color fmt s = Cli.format_with_style [color] fmt s in
```

Comment:
This is lacking a `Format.pp_print_as`  (like `punctuation`) which leads to Format column calculation errors when used with unicode chars.